paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -174,6 +174,7 @@ internal fun AddCard(
             // TODO (jameswoo) add to lokalize
             label = "Add",
             isEnabled = true,
+            isLoading = viewState.isProcessing,
```

Comment:
Can we add a test for this?

Reply:
Will do when the add payment method is implemented